app/src/main/java/ch/epfl/sdp/contamination/ConcretePositionAggregator.java

```diff
@@ -152,4 +152,9 @@ public void updateToOffline() {
     public void updateToOnline() {
         this.isOnline = true;
     }
+
+    @Override
+    public void stopAggregator() {
+        updatePosTimer.cancel();

```

Comment:
Why would one call it ? And when does it start again ?
As I understand, we call stopAggregator() when you have no GPS connection, but where does it start again ?

Reply:
Ohh, leftover function

Reply:
as you said